cypress/integration/inventoryPage.js

```diff
@@ -0,0 +1,43 @@
+const {it} = require('mocha');
+
+describe('sorting tests', function () {
+    //Logging into site
+    this.beforeEach(function () {
+        cy.login('standard_user', 'secret_sauce');
+    });
+    it('should sort all item names in ascending order', function () {
+        cy.InventorySortItems('Name (A to Z)');
```

Comment:
all methods should be named from the smaller letter: cy.loginAsAUser, cy.sortItems

Btw, is there a reason to use inventory everywhere?

Reply:
Fixed

Reply:
Fixed